public/js/graph/react_graph.js

```diff
@@ -1347,5 +1333,36 @@ var InfoBox = React.createClass({
         }
     }
 });
-
+var Modal = React.createClass({
+    getInitialState: function () {
+        return {
+            courseId: '',
+            course: [],
+            sessions: []
+            modalIsOpen: false,
+            courseTitle : '',
+            formatCourse : ''
+        };
+    },
+    closeModal : function() {
+        this.setState({modalIsOpen: false});
+        },
+    render: function () {
+            return (
+                <div>
+                <ReactModal className='ModalClass'
+                    overlayClassName='OverlayClass'
+                    isOpen={this.state.modalIsOpen}
+                    onRequestClose={this.closeModal}>
+                    <div className='modal-header'>
+                        {this.state.courseTitle}
+                          </div>
+                        <div className='modal-body'>
+                            <Description course  = {this.state.formatCourse}/> </div>  
+                </ReactModal>
+                </div>
+            );
+        }
+    }
```

Comment:
There is an extra `}` here. You would have caught this error if you fixed the indentation of your code.